File: fundamentals/easy/descendingOrder.ts
function descendingOrder(n: number) {
	return n.toString().length === 1
		? n
		: Number(
				n
					.toString()
					.split('')
					.sort((a, b) => Number(b) - Number(a))
					.join('')
		  )
}

export { descendingOrder }

File: codewars/julia/easy/descendingOrder.jl
function descendingOrder(n::Int)
  nStr = string(n)
  nStrArr = split(nStr, "")
  nIntArr = map(x -> parse(Int, x), nStrArr)
  nIntArrSorted = sort(nIntArr, rev=true)

  return parse(Int, join(nIntArrSorted, ""))
end


println(descendingOrder(42145))
